### DISCUSSION TITLE
有测过推理耗时吗，我测试下来和UniMERNet耗时差不多？

### DISCUSSION THREAD

**Discussion by limaopeng1** [SWHL/TrOCR-Formula-Rec#4] [Q&A]

**Comment by SWHL** [comment]
你测的那个？我用的是`microsoft/trocr-small-stage1`

**Comment by limaopeng1** [comment]
用的也是microsoft/trocr-small-stage1，图片是UniMER-Test/cpe/0000000.png

测试generate的耗时，先让模型跑了几次，然后才计算这张图片的推理耗时，这张图片的耗时是2s左右，和UniMERNet差不多，但UniMERNet毕竟是一个差不多4G的模型，这是哪里有问题吗？
s3 = time.perf_counter()
generated_ids = model.generate(pixel_values)
s4 = time.perf_counter()

**Reply by SWHL** [reply]
我还没有单测过耗时，看着没啥问题呀。

**Reply by limaopeng1** [reply]
你用eval.py跑整个UniMER-Test需要多长时间？我跑UniMER-Test十多个小时都还没跑完。。。

**Reply by SWHL** [reply]
我这在跑测试，的确很慢，也要好几个小时了。。。。。。。

**Comment by limaopeng1** [comment]
UniMERNet用的是这个模型：https://huggingface.co/wanderkid/PDF-Extract-Kit/tree/main/models/MFR/UniMERNet
同张图片耗时大概是2.8s

**Comment by SWHL** [comment]
有兴趣一起搞吗？

**Reply by limaopeng1** [reply]
可以啊，能给个微信或者别的？

**Reply by SWHL** [reply]
你留个邮箱，我邮箱发你

**Reply by limaopeng1** [reply]
我发你Oerview上留的163邮箱了

**Reply by SWHL** [reply]
收到，已经加了
